recipe/meta.yaml

```diff
@@ -1,37 +1,34 @@
 {% set name = "openvino" %}
-{% set version_postfix = "2024.4.0" %}
-{% set version = "2024.4.0" %}
+{% set version_postfix = "2024.5.0" %}
+{% set version = "2024.5.0" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
   - url: https://github.com/openvinotoolkit/openvino/archive/refs/tags/{{ version }}.tar.gz
-    sha256: aeb6297e3e88b4ab7803b27b231a72bd0493b6761ef98f225142146be500f538
+    sha256: 3d5b89760e1c946aca7b219dff4bf4cf5eb511932863c90cafc47d3c01c85744
     patches:
     - patches/001-protobuf-version.patch
```

Comment:
we need to drop this patch as it's fixed here https://github.com/openvinotoolkit/openvino/commit/79b0baddd05652b88092fccd9cb1c14846ea77a3#diff-a384efac67b1ca1369c48d6daa51cf43dddd459c2d578ce890f1cad632664e92